runtime/path_convert/src/main.cpp

```diff
@@ -170,18 +175,17 @@ static const test_data datas[] = {
     ,{"foo:echo /bar/baz", "foo:echo /bar/baz", false}
     ,{"@/foo/bar", "@" MSYSROOT2 "/foo/bar", false}
     ,{"@/foo/bar@", "@" MSYSROOT2 "/foo/bar@", false}
-    ,{"'@/foo/bar'", "'@" MSYSROOT2 "/foo/bar'", false}
+    ,{"'@/foo/bar'", "'@/foo/bar'", false}
     ,{"///foo/bar", "//foo/bar", false}
-    ,{".:./", ".;.\\", false}
-    ,{"..:./", "..;.\\", false}
-    ,{"../:./", "..\\;.\\", false}
-    ,{"../:./", "..\\;.\\", false}
-    ,{"../aa/:./", "..\\aa\\;.\\", false}
+    ,{".:./", ".:./", false}
+    ,{"..:./", "..:./", false}
+    ,{"../:./", "../:./", false}
+    ,{"../aa/:./", "../aa/:./", false}
     ,{"../", "../", false}
     ,{"/foo/bar/", "" MSYSROOT2 "/foo/bar/", false}
     ,{"-MExtUtils::ParseXS=process_file", "-MExtUtils::ParseXS=process_file", false}
     ,{"/foo/bin/../libs", "" MSYSROOT2 "/foo/bin/../libs", false}
-    ,{"'/foo/bin/../libs'", "'" MSYSROOT2 "/foo/bin/../libs'", false}
+    ,{"'/foo/bin/../libs'", "'/foo/bin/../libs'", false}
```

Comment:
This concerns me, too. That test shows behavior I would expect, and the new behavior looks wrong to me. :-(

Reply:
Same thing here, eh? There's a leading `'`.